### ISSUE TITLE
Detepction performance degenerates after finetuning

### ISSUE THREAD

**Issue by marialeyvallina** [QwenLM/Qwen2.5-VL#776]
I have been trying to adapt Qwen2.5-VL-7B to my own custom dataset, and I am currently experiencing an issue of degeneration of the output after finetuning.
The model off-the-shelf seems to work acceptably fine (average IOU around 0.20, not very good, but not disastrous.)
After finetuning the performance drops in half, to mean IOU of around 0.10.
The format of the messages I am using for finetuning looks like this one:
```
{"messages": [
                        {
                            "role": "user",
                            "content": [
                                {
                                    "type": "text",
                                    "text": "Locate the {object_name}, report the bbox coordinates in JSON format.",
                                },{
                                    "type": "image",
                                    "image": f"file://{impath}",
                                }
                            ],
                        },
                        {
                            "role": "assistant",
                            "content": [{"type": "text", "text":  ```json
                                  [
                                  	{"bbox_2d": [x0, y0, x1, y1], "label": {object_name}"}
                                  ]
                                  ```}],
                        },
                    ],
                    }    
```

The format of the bounding boxes is in absolute coordinates after passing the image by the Qwen2.5-VL processor, such as:
```
    text = processor.apply_chat_template(
        messages_before_bboxes, tokenize=False, add_generation_prompt=True
    )
    img_inputs, vid_inputs = process_vision_info(messages_before_bboxes)


    inputs = processor(
        text=text,
        images=img_inputs,
        videos=vid_inputs if vid_inputs else None,
        padding=True,
        return_tensors="pt",
    )
    raw_bbox =  eval(raw_bbox) # original format of the bboxes is relative, between 0 and 1
    xs = raw_bbox[::2]
    ys = raw_bbox[1::2]
    input_height = (inputs['image_grid_thw'][0][1]*14).item()
    input_width = (inputs['image_grid_thw'][0][2]*14).item()
    xmin = int(min(xs)*input_width)
    ymin = int(min(ys)*input_height)
    xmax = int(max(xs)*input_width)
    ymax = int(max(ys)*input_height)
    bbox = str([xmin, ymin, xmax, ymax])
```

Would you have any feedback about what I might be doing wrong so that the detection performance degenerates after training? I followed the same pipeline with Qwen2 (except there the coordinates were normalized between 0 and 100), and it trained successfully.

Thank you!

**Comment by LiGuo12** [comment]
Did you load model in torch.float16? it may decrease the preformence. It's better to load in bfloat16 or float32.

**Comment by shiyongde** [comment]
same problem， finetune qwen2.5vl got MAP 0.0 ！
